apps/chat/tests/test_routing.py

```diff
@@ -25,13 +23,11 @@
 def test_experiment_routing(with_default, routing_response, expected_tag):
     experiment = _make_experiment_with_routing(with_default)
     session = ExperimentSessionFactory(experiment=experiment)
-    fake_llm = FakeLlm(responses=[routing_response, "How can I help today?"], token_counts=[0])
-    fake_service = FakeLlmService(llm=fake_llm)
-    with patch("apps.experiments.models.Experiment.get_llm_service", new=lambda x: fake_service):
+    with mock_experiment_llm_service(["safe", "How can I help today?"], token_counts=[1]) as service:
```

Comment:
thought for separate pr but I've seen mock conversations like this required a lot in tests, curiuos if we can extract to a different file and import in some way?

Reply:
I'm curious how that would look. The `mock_experiment_llm_service` is already doing the heavy lifting and it's often necessary to customize the messages and token counts (I could have left the `token_counts` arg out since the value I'm providing is the same as the default).

But I'm all for reducing code duplication so keen to hear your ideas.

Reply:
true it would only be helpful if those were consistent so I'll look through previous tests for more concrete examples and if it would actually be useful 👍